# Goal
string tool_name        # 'name' in suction_tools.yaml
bool   turn_suction_on  #  True: on, False: off
bool   eject            #  True: expel air, False: do nothing
---
# Result
bool   success
---
# Feedback
uint32 suctioned        # digital_input of ur_control
